modules/upload-definitions-to-central-api-token.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * configuration/enable-offline-mode.adoc
+:_module-type: PROCEDURE
+[id="upload-definitions-to-central-api-token_{context}"]
+= Uploading definitions to Central by using an API token
+
+You can upload the vulnerability definitions database (that Scanner uses) to Central by using an API token.
+
+.Prerequisites
+
+* You must have an API token with the administrator role.
+* You must have installed the `roxctl` command-line interface (CLI).
+
+.Procedure
+
+. Set the `ROX_API_TOKEN` and the `CENTRAL_ADDRESS` tokens:
++
+[source,terminal]
+----
+$ export ROX_API_TOKEN=<api_token>
+----
++
+[source,terminal]
+----
+$ export CENTRAL_ADDRESS=<address>:<port_number>
+----
+. Run the following command to upload the definitions file:
++
+[source,terminal]
+----
+$ roxctl scanner upload-db \
+  -e "$CENTRAL_ADDRESS" \
+  --scanner-db-file=<compressed_scanner_definitions.zip>
+----
+
+[role="_additional-resources"]
+== Additional resources
```

Comment:
Same thing, this should be moved to the assembly since you can't have it in the module.

Also, if it's supposed to follow a module, it should use the `.Additional resources` format